readme.md

```diff
@@ -33,6 +33,12 @@ $ de --help
 
 It's `de` and not `del`, since `del` is already a builtin command on Windows.
 
+If preferred, a simple alias could fix this of course.
```

Comment:
Okay, this is nitpicking, but I don't think that *fixes* the problem, because AFAIK Windows doesn't support `alias`.